src/main/java/nl/novi/techiteasy/mappers/TelevisionMapper.java

```diff
@@ -30,6 +32,15 @@ public TelevisionDto toTelevisionDto(Television television) {
         dto.setOriginalStock(television.getOriginalStock());
         dto.setSold(television.getSold());
         dto.setLastSold(television.getLastSold());
+        if (television.getRemoteController() != null) {
+            dto.setRemoteController(RemoteControllerMapper.toRemoteControllerDto(television.getRemoteController()));
```

Comment:
Waarom heb je deze "toRemoteControllerDto()" methode static gemaakt? De RemoteControllerMapper is een `@`Component, wat betekend dat je het via constructor injection in deze TelevisionMapper beschikbaar kunt maken en op die manier de methode kunt aanspreken. Het is af te raden om dit dan nog static te maken